### DISCUSSION TITLE
How to enabled last triggered on vertical card

### DISCUSSION THREAD

**Discussion by TheZoker** [UI-Lovelace-Minimalist/UI#477] [Q&A] [answered]
Hi there,

First: Amazing project, really love the design!

I saw this vertical button example on the dev section of the docs:
![image](https://user-images.githubusercontent.com/1368405/155862957-7cadb594-5d83-45da-8455-a5697f3d49d2.png)

And I was wondering how I can archive this. So how can I enable the "last triggered" info at the bottom?

**Comment by TheZoker** [comment, marked as answer]
Just figured it out:
The `show_last_changed: true` needs too be set (currently missing on the documentation page)

**Comment by sildehoop** [comment]
If jou want you can make an pr or an issue so we can add it to de documentation it😄

**Reply by TheZoker** [reply]
Yes will do :)
